.circleci/config.yml

```diff
@@ -0,0 +1,71 @@
+version: 2.1
+
+executors:
+  mac-executor:
+    macos:
+      xcode: "11.2.1"
+    working_directory: ~/code
+
+jobs:
+  install_dependendies:
+    executor: mac-executor
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - gem-cache-v1-{{ checksum "Gemfile.lock" }}
+            - gem-cache-v1
+      - run:
+          name: Bundler
+          command: |
+            bundle install --path vendor/bundle --jobs 4 --retry 3
+      - save_cache:
+          key: gem-cache-v1-{{ checksum "Gemfile.lock" }}
+          paths:
+            - vendor/bundle
+      - persist_to_workspace:
+          root: .
+          paths:
+            - .
+  test:
+    executor: mac-executor
+    steps:
+      - checkout
+      - attach_workspace:
```

Comment:
As mentioned, rather than doing this can we just restore the bundler cache here?